src/components/sbb-panel/sbb-panel.stories.js

```diff
@@ -1,17 +1,88 @@
 import { h } from 'jsx-dom';
 import readme from './readme.md';
+import getMarkupForSvg from '../../global/helpers/get-markup-for-svg';
 
-const Template = (args) => <sbb-panel {...args} />;
+/* ************************************************* */
+/* Slot templates, used in Storybook template        */
+/* ************************************************* */
 
-export const sbbPanel = Template.bind({});
+/* --- text ---------------------------------------- */
 
-sbbPanel.args = {
-  'button-text': 'Sample button text',
-  text: 'Sample panel text',
+const panelText = {
+  control: {
+    type: 'text',
+  },
+  table: {
+    category: 'text',
+  },
+};
+
+/* --- text slot ----------------------------------- */
+
+const sbbTextArgTypes = {
+  text: panelText,
+};
+const sbbTextArgs = {
+  text: 'Rücksichtsvoll mit SBB Green Class',
+};
+
+const SlotTemplateSbbText = (args) => <span slot="text">{args.text}</span>;
+
+/* --- link slot ----------------------------------- */
+
+const sbbLinkArgs = {
+  href: 'https://www.sbb.ch/',
+  text: 'Mehr erfahren',
+};
+
+const SlotSbbLinkTemplate = (args) => (
+  <sbb-link
+    slot="link"
+    href-value={args.href}
+    icon="chevron-small-right-small"
+    icon-flip=""
+    icon-placement="end"
+    id-value=""
+    text={args.text}
+    text-size="m"
+    variant="block-negative"
+  >
+    <span slot="icon">
+      <span slot="icon">{getMarkupForSvg('chevron-small-right-small')}</span>
+    </span>
```

Comment:
Should we not control this also programmatically in the component, so never a wrong icon can be chosen? But better wait for sbb-icon component.